samples/widgets/textctrl.cpp

```diff
@@ -64,6 +64,7 @@ enum
 
     TextPage_Password,
     TextPage_NoVertScrollbar,
+    TextPage_NoBorder,
```

Comment:
I don't think we need to modify the sample, it already has "Set border" submenu which should allow testing with setting any border style for any of the controls, including the text control.